.circleci/config.yml

```diff
@@ -61,8 +68,10 @@ jobs:
       - persist_to_workspace:
           root: ~/project
           paths:
+            - config_files/secret
```

Comment:
means we can access the generated config files in later steps in circle - useful if theres a docker layer cache miss